data/objects/food_item.urdf

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+<robot name="food_item">
+  <link name="food_item">
+    <!--collision>
```

Comment:
Why is this commented out?

Reply:
Is there a possibility we use `food_item.urdf` with collision shape? If so, we could programmatically disable collision of this item through DART.

Here is an example code:
```cpp
body->getShapeNode(0)->getCollisionAspect()->setCollidable(false);
```

This will entirely remove this from being collision checked. If we only want to disable for an object pair (e.g., the fork and food item), then we could use collision filter.

Reply:
Yes sounds good, worth trying out 👍

Reply:
@Tonkel Could you uncomment the collision? Thanks!

Reply:
yup, will do that.

Reply:
done